packages/core/strapi/lib/types/core/config/plugin.d.ts

```diff
@@ -0,0 +1,9 @@
+export interface PluginConfigs {
+  [key: string]:
+    | {
+        enabled: boolean;
+        resolve?: string;
+        config?: object;
+      }
+    | boolean;
```

Comment:
Didn't know it could be just true, TIL

Reply:
well yes if it doesn't need configs, it can be just enabled